integration/class-surge.php

```diff
@@ -177,11 +171,20 @@ public static function test_surge_integration() {
 				\sprintf(
 					// translators: %s: Plugin directory path.
 					\__( 'To enable the ActivityPub integration with Surge, add the following line to your <code>wp-config.php</code> file: <br /><code>%s</code>', 'activitypub' ),
-					self::$cache_config
+					self::cache_config()
 				)
 			);
 		}
 
 		return $result;
 	}
+
+	/**
+	 * Get the cache config.
+	 *
+	 * @return string The cache config.
+	 */
+	public static function cache_config() {
```

Comment:
```suggestion
	public static function get_cache_config() {
```

what about using a getter?

Reply:
Fixed in [4a3020b](https://github.com/Automattic/wordpress-activitypub/pull/1696/commits/4a3020bedce313b8a1efa7d084d9993ebbed6b6b).